ntbs-service/wwwroot/css/home.scss

```diff
@@ -1,32 +1,26 @@
-.alerts-heading {
+.alerts-heading, .kpi-heading {

```

Comment:
Should we just give this a single class `heading`?
Also wondering if we actually need something like this, rather than using one of headings classes from NHS frontend https://beta.nhs.uk/service-manual/styles-components-patterns/typography#headings?